src/pages/index.js

```diff
@@ -15,6 +15,7 @@ export default function Home() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <main className={styles.main}>
+        <Hero />
         <section className={styles.listings_container}>
```

Comment:
@ibimina When I pulled your branch down and ran the app, I do not see this `<section>` code in my local copy.

Please update this branch with latest `main` thanks!

Reply:
okay